src/core/agent/services/signifyNotificationService.ts

```diff
@@ -603,6 +635,16 @@ class SignifyNotificationService extends AgentService {
                   .get(admitExchange.exn.p);
                 const credentialId = grantExchange.exn.e.acdc.d;
                 if (credentialId) {
+                  const notifications =
+                      await this.notificationStorage.findAllByQuery({
+                        exnSaid: grantExchange.exn.d,
+                      });
+                  for (const notification of notifications) {
+                    Agent.agent.signifyNotifications.deleteNotificationRecordById(
```

Comment:
Can you check this with single-sig identifier too? I'm pretty sure the notification will have already been deleted, so this will fail